docs/usage.md

```diff
@@ -1,5 +1,5 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|---------------------------------------------|
 
 # Usage
 
```

Comment:
Update the below line
>Line#10 >> Rephrase the sentence as per the new Label given to the field
>Update the image "custom-module.png"
>Update image "filter-for-single-module.png"
>Crop the Image "custom-module-layer.png" and remove the "Add Layer" button from bottom
>Rephrase the line #41 >> Avoid using the term "Custom module"
>In line#54, change the value "0" to "?" >> Also remove " (In this case for incident) "
>Line#61 > Refrain using the term "Multiple Modules" and use "Count of Records Across Module"

Reply:
Also Update the image "funnel.png"
![image](https://user-images.githubusercontent.com/24773177/234538290-554c68a9-1ec4-4be4-862f-541a9958dec0.png)